pkg/apis/nmstate/v1alpha1/zz_generated.openapi.go

```diff
@@ -147,7 +147,7 @@ func schema_pkg_apis_nmstate_v1alpha1_NodeNetworkConfigurationEnactmentStatus(re
 			},
 		},
 		Dependencies: []string{
-			"./pkg/apis/nmstate/v1alpha1.Condition", "./pkg/apis/nmstate/v1alpha1.State"},
+			"github.com/nmstate/kubernetes-nmstate/pkg/apis/nmstate/v1alpha1.Condition", "github.com/nmstate/kubernetes-nmstate/pkg/apis/nmstate/v1alpha1.State"},
```

Comment:
You have some GOFLAGS at your env, please unset them before call `make generate`